lib/bot.js

```diff
@@ -144,9 +144,18 @@ class Bot {
         this.ircClient.say(ircChannel, prelude);
         this.ircClient.say(ircChannel, text);
       } else {
-        text = `<${coloredUsername}> ${text}`;
-        logger.debug('Sending message to IRC', ircChannel, text);
-        this.ircClient.say(ircChannel, text);
+        if (text !== '') {
+          text = `<${coloredUsername}> ${text}`;
+          logger.debug('Sending message to IRC', ircChannel, text);
+          this.ircClient.say(ircChannel, text);
+        }
+        if (message.attachments) {
```

Comment:
So far I haven't found a way to get both into a single message. I'll do as you suggested!